services/core-web/common/actionCreators/noticeOfWorkActionCreator.js

```diff
@@ -88,6 +88,10 @@ export const importNoticeOfWorkApplication = (applicationGuid, payload) => (disp
       createRequestHeader()
     )
     .then((response) => {
+      notification.success({
+        message: "Successfully verified the Notice of Works location",
```

Comment:
**Notice of Work's**

This function would be better if it took the message as a parameter.